Dockerfile

```diff
@@ -43,3 +43,6 @@ RUN find ${HOME} -mindepth 1 -delete
 RUN mkdir -p /tmp/install_deps/patches/
 COPY hack/patches/* /tmp/install_deps/patches/
 RUN /tmp/install_deps/patches/apply_patches.sh
+
+# Install external resources needed for automated tagging
+RUN python3 -c 'import f8a_tagger; f8a_tagger.prepare()'
```

Comment:
Could we move this closer to the tagger installation, i.e. line 37 ?

Reply:
Sure, will do.

Reply:
Or we could add this directly to the install script.